### ISSUE TITLE
Any sample workflow ?

### ISSUE THREAD

**Issue by Metal-dude** [AustinMroz/ComfyUI-SpliceTools#1]
Hello Austin, 

I like the concept of your repo and I think there is a lot of potential, given the growing popularity of Sampler Custom.

Yet, I can't find any workflow anywhere. Could you add some sample workflows please ?

I'm also confused by Splice Latents : the description says :" Uses a Gaussian blur to approximately split a noised latent into noise and un-noised latents."

Given this description, one could expect 1 latent entry point and 2 latent exit points. Yet, this is the opposite : the node has 2 latent entry points and a single latent exit point.

Thanks.

**Comment by AustinMroz** [comment]
I agree that example workflows would be beneficial. I have a single prepared colorization example workflow and will provide an update with others later this evening.
![WinstonCurchillColorized](https://github.com/AustinMroz/ComfyUI-SpliceTools/assets/4284322/52fd9ad8-0a84-4341-aa2c-3c8a04d06f4a)

I also agree that the description for Splice Latents could be improved and will improve it adding the examples to the README. The node performs the splitting operation twice and then merges the lower details of one input with the higher details of another. If you merely wish to filter one side of the details, you can simply ommit the other input. As a rough example: workflow
![SpliceReference](https://github.com/AustinMroz/ComfyUI-SpliceTools/assets/4284322/d4be66b7-f20e-4927-a5da-07f1561694e9)

**Comment by Metal-dude** [comment]
Thank you. I understand better.

Your "Splice Latents" node enables to separate the color from the details of the image. 

When a latent is connected to the lower entry point, the node keeps the color, while blurring the details of the image.

And when a latent is connected to the upper entry point, the node keeps the details of the image, while omitting the color.

As a result, your node can be very useful to change the color of images.

**Comment by jags111** [comment]
awesome same request here to understand this well.
